.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,28 +21,30 @@
 -->
 
 
-## Known Issues and TODO List:
+
+### Tests Performed:
+<!-- Does it build without errors? Did you test in-game? What did you test? Did you do all these tests on Linux, Mac or Windows? Other tests performed -->
```

Comment:
Personally I prefer to have tests performed before how to test to so it is in order
1. What has been done
2. What needs to be done

Reply:
Done